addon/models/building-work.js

```diff
@@ -4,7 +4,7 @@ import Building from "./building";
 import XMLModel from "./xml-model";
 
 export default class BuildingWork extends XMLModel {
-  @tracked kindOfWork;
+  @tracked kindOfWork = BuildingWork.kindOfWorkOptions[0];
```

Comment:
Does the select of the validated form now show and use the default value correctly? I never actually looked into that issue last week.

Reply:
Hmm good question. I will test this really quick.

Reply:
Yep works :+1:

Reply:
Nice :ok_hand: